lib/compat/wordpress-6.1/blocks.php

```diff
@@ -71,12 +71,12 @@ function gutenberg_block_type_metadata_view_script( $settings, $metadata ) {
 		! isset( $metadata['viewScript'] ) ||
 		! empty( $settings['view_script'] ) ||
 		! isset( $metadata['file'] ) ||
-		! str_starts_with( $metadata['file'], gutenberg_dir_path() )
+		! str_starts_with( $metadata['file'], wp_normalize_path( gutenberg_dir_path() ) )
```

Comment:
First, here the two arguments have different delimiters: 
- `$metadata['file']`: `D:/path/to/public/wp-content/plugins/gutenberg/build/block-library/blocks/file/block.json`
- `gutenberg_dir_path()`: `D:\path\to\public\wp-content\plugins\gutenberg/`

So, replace `$metadata['file']` with forward slashes.